build.gradle.kts

```
@@ -35,6 +35,10 @@ tasks {
 }
 
 dependencies {
+  // openapi
+  implementation("org.springdoc:springdoc-openapi-ui:1.5.2")
```

_nitpick_
🙋‍♂️ There seems to be a newer patch version that was released recently
```suggestion
  implementation("org.springdoc:springdoc-openapi-ui:1.5.3")
```